### PR TITLE
[General Azerite] Blessed Portents trait module

### PR DESCRIPTION
**Pull request by Nalhan**
Issue #2242 
Back again with another azerite trait. Tracks Blessed Portents healing contribution and shows number of buffs applied, refreshed, procced, expired, and active at the end of the fight.
chrome_2018-09-25_16-11-23

### CODE REVIEW

File: src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/BlessedPortents.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.BLESSED_PORTENTS_HEAL.id) {
+      return;
+    }
+
+    this.healing += event.amount + (event.absorbed || 0);
+
+    //const targetId = event.targetID;
```
Comment (joshinat0r) [inline src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/BlessedPortents.js]: Please remove this.

File: src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/BlessedPortents.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      return;
+    }
+
+    this.refreshedBuffs++;
+  }
+  get totalBuffs() {
+    return this.proccedBuffs + this.expiredBuffs + this.refreshedBuffs + this.activeBuffs;
+  }
+
+   statistic() {
```
Comment (joshinat0r) [inline src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/BlessedPortents.js]: Indentation :)

**Comment by Nalhan** [comment]
Done. I guess I forgot to commit removing that comment because it was gone locally. Whoops.
I'll keep the example report thing in mind for next time; here's one I used for testing just now for posterity: https://www.warcraftlogs.com/reports/gBKALRbtPVYW6HyZ#fight=14&type=auras&sourceclass=Any&ability=271843&by=target

**Comment by joshinat0r** [comment]
Thanks 👍

**Pull request merged**